instagram-clone/src/components/PostContainer/PostLikes.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react'
+import styled from 'styled-components';
+
+ class PostLikes extends Component {
+
+    state = {
+        likes: this.props.postLike
```

Comment:
I hope it makes sense why we wouldn't want to separate our state into components like this. Updating our data here and only here means that our data in the rest of the app is out of sync. Let me know if we should go over this in more detail.